samples/MvcSample.Web/wwwroot/Scripts/Mvc.js

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+var Mvc = (function ($) {
+
+    // This holds the stringified result.
+    var result = "";
```

Comment:
:+1: closing over the 'result' seems really strange. Just declare it inside `_stringify` and pass it down.